### ISSUE TITLE
Multi-Language Support

### ISSUE THREAD

**Issue by andoriyaprashant** [litmuschaos/litmus-docs#271]
## Is this a BUG REPORT or FEATURE REQUEST?
 
FEATURE REQUEST

**What happened**:

The website currently lacks multi-language support, offering content only in English, which limits accessibility for non-English speaking users and restricts our global reach.

**What you expected to happen**:

Implement an option to switch languages on the website to enhance user accessibility and global reach.

**How to reproduce it (as minimally and precisely as possible)**:

![image](https://github.com/litmuschaos/litmus-docs/assets/121665385/57c6d375-bfe3-4caa-9e87-ccf6bf95d83b)
